console/src/app/pages/projects/apps/app-create/app-create.component.html

```diff
@@ -439,6 +463,14 @@ <h1>{{ 'APP.PAGES.CREATE_DESC_TITLE' | translate }}</h1>
         "
       >
         <div class="formfield full-width">
+          <mat-slide-toggle
+            color="primary"
+            name="devMode"
+            matTooltip="{{ 'APP.OIDC.DEVMODEDESC' | translate }}"
+            (change)="onChangeDevMode($event)"
```

Comment:
lets use [(ngModel)]="devMode" here so we need no change handler
probably we also have to add `[ngModelOptions]="{ standalone: true }"`